gprofiler/main.py

```diff
@@ -572,10 +573,17 @@ def parse_cmd_args() -> configargparse.Namespace:
         " as well",
     )
 
+    parser.add_argument(
+        "--collect-databricks-job-name",
```

Comment:
Just `--databricks-ephemeral-job`? This isn't just "collect job name", it's replacing the service name here